tools/bin/build_image.sh

```diff
@@ -8,6 +8,7 @@ DOCKERFILE="$3"
 TAGGED_IMAGE="$4"
 ID_FILE="$5"
 FOLLOW_SYMLINKS="$6"
+DOCKER_BUILD_ARCH="${DOCKER_BUILD_ARCH:-amd64}"
```

Comment:
would this work on x64 by default as well? i.e: is AMD64 the correct default for all architectures?

Reply:
It's the default value in all our Dockerfiles 
eg https://github.com/airbytehq/airbyte/blob/master/airbyte-workers%2FDockerfile#L4

It would be nice if someone with someone with an intel chip tried it before we merge to confirm though